tools/mochaRunner/src/utils.ts

```diff
@@ -76,14 +76,16 @@ export function printSuggestions(
         return item.expectation;
       })
     );
-    console.log(
-      'The recommendations are based on the following applied expectaions:'
-    );
-    prettyPrintJSON(
-      toPrint.map(item => {
-        return item.basedOn;
-      })
-    );
+    if (action !== 'remove') {
```

Comment:
why do we want to hide if the action is remove?

Reply:
We we want to remove Expectation A its based on Expectation A, no need to show it 2 times.